RobustGenericAttribution/README.md

```diff
@@ -0,0 +1,38 @@
+# Documentation
+
+The **RGA** (Robust Generic Attribution) standard is intended to be a flexible, open, and readable way <!--Insert more marketing bull that sounds good here!--> to define various metadata such as licensing and attribution for an arbitrary collection of files of many types such as sound effects or prototypes. An RGA file contains metadata for all of the files in the same directory as the RGA file (not including subdirectories). The entries in an RGA file contain specific metadata such as the author of the file(s) or a description of any modifications made to them (in compliance with many Creative Commons licenses).
+
+An RGA is a YAML file with the name `attributions.yml`, and contains an arbitrary number of entries as defined below.
+
+Typically whenever new files are being added, a new entry will be added as it likely won't contain the same metadata as existing entries. However, one may append files to an existing entry if the metadata is otherwise identical.
+
+## YAML
+
+An RGA file must be named `attributions.yml`. All values within entries are wrapped in double-quotes (`""`).
+
+The YAML contains an arbitrary number of entries, covering all files in the same directory as the RGA file. An entry is defined as follows:
+
+Key | Meaning
+--- | -------
+`files` | An array of filenames (with extensions) that this entry applies to. The filename order is arbitrary.
+`copyright` | The copyright holder and info for finding the source of the file(s).
+`license` | A valid [SPDX License Identifier](https://spdx.org/licenses/) applying to all files within an entry.
```

Comment:
`Resources/Audio/Effects/hit_kick.ogg` does not use a valid SPDX license identifier.
Instead, licenses text is as follows:
```
hit_kick.ogg is made by Taira Komori
(https://taira-komori.jpn.org/freesounden.html)
```
with the site stating:
```
Hi, welcome to my COMPLETELY FREE SOUND EFFECTS website.
I'm Taira Komori in Japan. I recorded and made all sounds by myself. These are mp3 and 160kbps.

Terms of Use
All sound effects on this website are created by Taira Komori.
You are allowed to use the sounds free of charge and royalty free in your projects (such as films, videos, games, presentations, animations, stage plays, radio plays, audio books, apps) be it for commercial or non-commercial purposes.

But you are not allowed to
- post the sounds on any website for others to download, copy or use
- sell, re-sell, license or re-license the sounds (as individual sound effects or as a sound effects library) to anyone else
- claim the sounds as yours
- link directly to individual sound files

If you use my sound effects, I wish you linking back to me on SNS or website.
"Let's Play with Free Sound Effects" https://taira-komori.jpn.org/freesounden.html

I recomend you checking my [Japanese free sound effects website](https://taira-komori.jpn.org/freesound.html) with Google chrome translation.
There are more sound effects. Thanks !
```

Reply:
Resolved.